java/src/jmri/jmrit/logixng/actions/ActionBundle.properties

```diff
@@ -382,7 +382,7 @@ ForEach_Long_Common                 = For each value, set variable "{1}" and exe
 ForEach_Long_LocalVariable          = For each value, set variable "{1}" and execute action {2}. Values from local variable {0}
 ForEach_Long_Memory                 = For each value, set variable "{1}" and execute action {2}. Values from memory {0}
 ForEach_Long_Formula                = For each value, set variable "{1}" and execute action {2}. Values from formula {0}
-ForEach_UserSpecifiedSource_Variable    = Variable
+ForEach_UserSpecifiedSource_Variable    = Local variable
```

Comment:
I assume that the Global variables also work.

If so, this exposes an interesting problem with LogixNG internal variable labels.  There are actually three labels, **Local** only, **Global** only or either.  For either, my inclination is **LogixNG variable**.

Reply:
> I assume that the Global variables also work.

Yes. The only case that's "local only" is the setup of local variables. And the only case of "global only" is the Global variable table.

When I created "Local variables", I had no thought of adding global variables.

There is 65 occurrencies of "Local variable" in the property files in `jmri.jmrit.logixng.**` packages. I think it's only two that are "Local only", the ones in `jmri\jmrit\logixng\tools\swing\LogixNGSwingBundle.properties`. Should the rest be renamed to `LogixNG variable`?

Reply:
So far no one has questioned the **Local variable** label.  Changing the property files is trivial but updating the help content is a lot more work.

Reply:
Sounds good to keep the **Local variable** labels as is.

What's your thought about this PR? Does it help or is it unnecessary?